### PR TITLE
pixie 0.8.3

### PR DESCRIPTION
**Pull request by ddelnano**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Uninstalling pixie... (10 files, 65.5MB)
  Error: Interrupted system call @ dir_s_rmdir - /usr/local/etc
  Error: Interrupted system call @ dir_s_rmdir - /usr/local/etc
```

File: Formula/p/pixie.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Pixie < Formula
   desc "Observability tool for Kubernetes applications"
   homepage "https://px.dev/"
   url "https://github.com/pixie-io/pixie.git",
-      tag:      "release/cli/v0.8.2",
-      revision: "401c92c1ed086eaa35dbe17b4fbcfd96e7b1ca4f"
+      tag:      "release/cli/v0.8.3",
+      revision: "3a76a2723ba050fdd9908ef41d04d5cac0075cc0"
```
Comment (chenrui333) [inline Formula/p/pixie.rb:6]:
```suggestion
      revision: "a4b8bc53eb1c44bed3753c6706325a27c28d93f0"
```

**Comment by chenrui333** [comment]
@ddelnano looks like the release build failed, https://github.com/pixie-io/pixie/actions/runs/10745392806/job/29804324458

**Comment by ddelnano** [comment]
I'll be opening a new PR since we just got the release build working. Apologies for opening the original one too early!

**Pull request closed without merging**